mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -71,6 +71,22 @@ private[recommendation] trait ALSModelParams extends Params with HasPredictionCo
 
   /** @group getParam */
   def getItemCol: String = $(itemCol)
+
+  /**
+   * Param for strategy for dealing with unknown users or items at prediction time.
+   * Supported values:
+   * - "nan": prediction value for unknown ids will be NaN.
+   * - "drop": rows in the input DataFrame containing unknown ids will be dropped.
+   * Default: "nan".
+   * @group expertParam
+   */
+  val unknownStrategy = new Param[String](this, "unknownStrategy",
```

Comment:
How about "coldStartStrategy"? For people familiar with recommendations "cold start" is clear in meaning. But it may be less clear for people less familiar.

@srowen what do you think?